setup.py

```diff
@@ -1,34 +1,70 @@
 """
-Setup module for trustymail
+setup module for trustymail
 
 Based on:
 
-- https://github.com/dhs-ncats/pshtt
+- https://packaging.python.org/distributing/
+- https://github.com/pypa/sampleproject/blob/master/setup.py
 """
 
 from setuptools import setup
+from trustymail import __version__
 
 setup(
+    name='trustymail',
 
-        name='trustymail',
-        version='0.0.1',
-        description="Scan DNS records for best mail practices.",
+    version=__version__,
+    description='Scan domains and return data based on trustworthy email best practices',
 
-        download_url="https://github.com/dhs-ncats/trustymail",
+    # NCATS "homepage"
+    url='https://www.dhs.gov/cyber-incident-response',
+    # The project's main homepage
+    download_url='https://github.com/dhs-ncats/trustymail',
 
-        packages=['trustymail'],
+    # Author details
+    author='Department of Homeland Security, National Cybersecurity Assessments and Technical Services team',
+    author_email='ncats@hq.dhs.gov',
 
-        install_requires=[
-            'requests',
-            'docopt',
-            'py3dns',
-            'pyspf==2.0.11',
-            'publicsuffix'
-        ],
+    license='License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
 
-        entry_points={
-            'console_scripts': [
-                'trustymail = trustymail.cli:main'
+    # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 4 - Beta',
+
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+
+        # Pick your license as you wish (should match "license" above)
+        'License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication',
+
+        # Specify the Python versions you support here. In particular, ensure
+        # that you indicate whether you support Python 2, Python 3 or both.
+        'Programming Language :: Python :: 3',
```

Comment:
Is this project exclusively Python 3, or would we intend to include Python 2 support as well?

Reply:
Based on the results I just saw when trying to run tox, the answer is Python 3 only, due to the `py3dns` dependency.

```
Collecting py3dns (from trustymail==0.0.1)
  Downloading py3dns-3.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/vr/7lfgcgzd0wb4brywdhg86ygw00157w/T/pip-build-V8AkXy/py3dns/setup.py", line 7, in <module>
        import DNS
      File "/private/var/folders/vr/7lfgcgzd0wb4brywdhg86ygw00157w/T/pip-build-V8AkXy/py3dns/DNS/__init__.py", line 27, in <module>
        from .Base import DnsRequest
      File "/private/var/folders/vr/7lfgcgzd0wb4brywdhg86ygw00157w/T/pip-build-V8AkXy/py3dns/DNS/Base.py", line 9
    SyntaxError: Non-ASCII character '\xc2' in file /private/var/folders/vr/7lfgcgzd0wb4brywdhg86ygw00157w/T/pip-build-V8AkXy/py3dns/DNS/Base.py on line 10, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Reply:
That's correct, Python 3 only.